dcm4che-net-audit/src/main/java/org/dcm4che3/net/audit/SyslogProtocolHandler.java

```diff
@@ -212,8 +246,12 @@ public void run() {
                                 s);
                         break;
                     }
-                    LOG.info("Received Syslog message of {} bytes from {}",
```

Comment:
Please restore the log.info messages we shouldn't be changing these to debug in dcm4che master.